app/javascript/components/Greetings.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+const Greetings = () => {
+  const [data, setData] = useState({}); // 1
+  const fetchData = async () => {
+    const result = await axios.get("http://localhost:3000/greetings");
+    setData(result.data);
+  };
+  console.log(data);
+  useEffect(() => {
+    fetchData();
+  }, []);
```

Comment:
- [ ] It will be great to have redux implemented in this project. Remember there should be a store a reducer and an action of fetching the greetings. In your implementation, you have left out the redux part. Kindly fix this